bluepysnap/nodes.py

```diff
@@ -239,7 +248,8 @@ def _positional_mask(self, node_ids):
         if node_ids is None:
             return np.full(len(self._data), fill_value=True)
         mask = np.full(len(self._data), fill_value=False)
-        mask[node_ids] = True
+        valid_node_ids = pd.Index(utils.ensure_list(node_ids)).intersection(self._data.index)
```

Comment:
Should be reflected in unit test (I'll check).

Reply:
this is done already:
```
npt.assert_equal(_call({"node_id": 1}), [1])
npt.assert_equal(_call({"node_id": [1]}), [1])
npt.assert_equal(_call({"node_id": [1, 2]}), [1, 2])
npt.assert_equal(_call({"node_id": [1, 2, 42]}), [1, 2])
```